### ISSUE TITLE
[JENKINS-66130] Improve Green Hills MULTI regular expression when columns are reported 

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/analysis-model#1383]
<p>When using the MULTI compiler option <b>--brief_diagnostics</b> the warnings plugin reports only one warning that contains the whole content. If there are many warnings the warnings overviews are missing completely (probably something crashes?). The main difference is that the warnings are single line warnings and they contain the column</p>

<p>Regular output:</p>

```
"D:\jenkins\src\MyFile.cpp", line 42: warning #381-D: extra ";" ignored

```
<p>...</p>

<p>Brief diagnostics:</p>

```
"D:\jenkins\src\MyFile.cpp", line 42 *(col. 58)*: warning #381-D: extra ";" ignored

```

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=itchify">itchify</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-66130" target="_blank">Improve Green Hills MULTI regular expression when columns are reported </a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/jagrutiti.png" title="jagrutiti's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jagrutiti">jagrutiti</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Major
<li><b>component(s)</b>: analysis-model
<li><b>label(s)</b>: hacktoberfest, help-wanted, newbie-friendly
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2023-02-07T10:38:32+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 3
<li><b>imported</b>: 20260108-221235
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>When using the MULTI compiler option <b>--brief_diagnostics</b> the warnings plugin reports only one warning that contains the whole content. If there are many warnings the warnings overviews are missing completely (probably something crashes?). The main difference is that the warnings are single line warnings and they contain the column</p>

<p>Regular output:</p>
<div class="preformatted panel" style="border-width: 1px;"><div class="preformattedContent panelContent">
<pre>"D:\jenkins\src\MyFile.cpp", line 42: warning #381-D: extra ";" ignored
</pre>
</div></div>
<p>...</p>

<p>Brief diagnostics:</p>

<div class="preformatted panel" style="border-width: 1px;"><div class="preformattedContent panelContent">
<pre>"D:\jenkins\src\MyFile.cpp", line 42 *(col. 58)*: warning #381-D: extra ";" ignored
</pre>
</div></div></pre>
</details>
</details>
<details><summary><i>environment</i></summary>

```
Warnings NG Plugin 9.2.<br/>
Analysis Model API Plugin 10.2.5
```
</details>
